app/src/js/reducers/sort-persist.js

```diff
@@ -0,0 +1,15 @@
+import { createReducer } from '@reduxjs/toolkit';
+import {
+  SORTS,
+} from '../actions/types';
+
+export const initialState = {
+  sorted: false,
+  maintained: false,
+};
+export default createReducer(initialState, {
+  [SORTS]: (state, action) => {
+    state.sorted = true;
+    state.maintained = action.holdPreviousState;
```

Comment:
We want the reducer to do something like this to the state

```suggestion
  [SORTS]: (state, action) => {
    state[action.table]sortBy = action.sortBy
```

From the table, sortBy looks like this:

```
[
	{
		id: [whateverfield is sorted],
		desc: true/false
	}
]
```

So ideally then, our state will look like this:
```
SORTS: [
	[table]: sortByArray
]

That way, on each table, we can preserve the state and then when we query the state, we can do
sort.collections for example on the collections page to get the sortBy value.